ChangeLog

```diff
@@ -41,17 +41,6 @@ New deprecations
    * Direct manipulation of structure fields of RSA contexts is deprecated.
      Users are advised to use the extended RSA API instead.
 
-API Changes
-   * Extend RSA interface by multiple functions allowing structure-
-     independent setup and export of RSA contexts. Most notably,
-     mbedtls_rsa_import and mbedtls_rsa_complete are introduced for setting
-     up RSA contexts from partial key material and having them completed to the
-     needs of the implementation automatically. This allows to setup private RSA
-     contexts from keys consisting of N,D,E only, even if P,Q are needed for the
-     purpose or CRT and/or blinding.
-   * The configuration option MBEDTLS_RSA_ALT can be used to define alternative
-     implementations of the RSA interface declared in rsa.h.
-
```

Comment:
I'm not sure the rebasing went so well. I wouldn't expect to see the `API Changes` section moved if based on a recent enough branch (one that has 2.7.0 in the ChangeLog, at least).

Reply:
It was rebased to some point during 2.7 development. We'll merge the changelog when we merge the PR, as usual, that's not a problem.